meta/runtime.yml

```
@@ -332,6 +332,11 @@ plugin_routing:
       deprecation:
         removal_version: 3.0.0
         warning_text: see plugin documentation for details
+    firewalld:
+      deprecation:
+        removal_version: 0.2.1
```

```suggestion
        removal_version: 2.0.0
```
(This is when we will remove the redirect.)